jwplatform/client.py

```diff
@@ -57,13 +47,13 @@ class Client(object):
         >>> jwplatform_client = jwplatform.Client('API_KEY', 'API_SECRET')
     """
 
-    def __init__(self, key, secret, *args, **kwargs):
+    def __init__(self, key: str, secret: str, *args, **kwargs):
         self.__key = key
         self.__secret = secret
 
         self._scheme = kwargs.get('scheme') or 'https'
         self._host = kwargs.get('host') or 'api.jwplatform.com'
```

Comment:
Unrelated to this update, but why are we not adding more arguments instead of trying to get them from the `kwargs` argument. It would give an IDE better hints if we just name the arguments.

Reply:
100%. I was thinking doing that in a follow up PR.